vmware/vcd-ext-sdk
# Refactor @vcd/sdk

**Migrate the project to angular library structure**
By migrating the project to angular library the maintaining
of the package will become way easier, all features supported
by angular are supported.

**Deprecate the i18n module located in @vcd/sdk**
Because of the migration the old implementation can't be
compiled and since we want to migrate to @vcd/i18n package
we can deprecate @vcd/sdk i18n package.

**Implement Angular plugin builder**
The angular builders are one of the angular's features,
this builder is capable to bundle any module, in container
plugin dev env to an actual plugin.

**Testing**
- Build plugin seed which using the new version on the module
- Verify the plugin works
- Verify the transllations can be activated even when @vcd/sdk
doesn't provide this module. Every plugin is capable to install
and inject it's i18n module.
